### PR TITLE
update beats: fix new file location

### PR DESCRIPTION
**Pull request by v1v**
## Motivation/summary

https://github.com/elastic/apm-server/pull/6287 moved the files to the `legacy` folder, lets update the scripts

### CODE REVIEW

File: .ci/scripts/update-beats.sh [hunk 1]
```diff
@@ -10,8 +10,8 @@ COMMIT_MESSAGE="Update to elastic/beats@$(go list -m -f {{.Version}} github.com/
 
 git checkout -b "update-beats-$(date "+%Y%m%d%H%M%S")"
 git add --ignore-errors go.mod go.sum NOTICE.txt \
-	.go-version docs/version.asciidoc \
-	docs/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go
+	.go-version docs/legacy/version.asciidoc \
+	docs/legacy/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go
```
Comment (axw) [inline .ci/scripts/update-beats.sh:14]:
```suggestion
	.go-version docs/version.asciidoc
```

```suggestion
	.go-version docs/version.asciidoc \
```

- version.asciidoc is still in docs, not docs/legacy
- fields.asciidoc _is_ in docs/legacy, but we're no longer updating it or generating include/fields.go. The integration package is the way forward, and apm-server no longer installs index templates, etc.

File: Makefile [hunk 2]
```diff
@@ -144,7 +144,7 @@ update-beats-module:
 	$(GO) get -d -u $(BEATS_MODULE)@$(BEATS_VERSION) && $(GO) mod tidy
 	cp -f $$($(GO) list -m -f {{.Dir}} $(BEATS_MODULE))/.go-version .go-version
 	find . -maxdepth 2 -name Dockerfile -exec sed -i'.bck' -E -e "s#(FROM golang):[0-9]+\.[0-9]+\.[0-9]+#\1:$$(cat .go-version)#g" {} \;
-	sed -i'.bck' -E -e "s#(:go-version): [0-9]+\.[0-9]+\.[0-9]+#\1: $$(cat .go-version)#g" docs/version.asciidoc
+	sed -i'.bck' -E -e "s#(:go-version): [0-9]+\.[0-9]+\.[0-9]+#\1: $$(cat .go-version)#g" docs/legacy/version.asciidoc
```
Comment (axw) [inline Makefile:147]:
```suggestion
	sed -i'.bck' -E -e "s#(:go-version): [0-9]+\.[0-9]+\.[0-9]+#\1: $$(cat .go-version)#g" docs/version.asciidoc
```

File: magefile.go [hunk 3]
```diff
@@ -45,7 +45,7 @@ func init() {
 	mage.SetBuildVariableSources(&mage.BuildVariableSources{
 		BeatVersion: filepath.Join(repo.RootDir, "cmd", "version.go"),
 		GoVersion:   filepath.Join(repo.RootDir, ".go-version"),
-		DocBranch:   filepath.Join(repo.RootDir, "docs/version.asciidoc"),
+		DocBranch:   filepath.Join(repo.RootDir, "docs/legacy/version.asciidoc"),
```
Comment (axw) [inline magefile.go:48]:
```suggestion
		DocBranch:   filepath.Join(repo.RootDir, "docs/version.asciidoc"),
```

**Review by axw — CHANGES_REQUESTED** [review]

File: .ci/scripts/update-beats.sh [hunk 4]
```diff
@@ -11,7 +11,6 @@ COMMIT_MESSAGE="Update to elastic/beats@$(go list -m -f {{.Version}} github.com/
 git checkout -b "update-beats-$(date "+%Y%m%d%H%M%S")"
 git add --ignore-errors go.mod go.sum NOTICE.txt \
 	.go-version docs/version.asciidoc \
```
Comment (v1v) [inline .ci/scripts/update-beats.sh:13]:
```suggestion
	.go-version docs/version.asciidoc
```

**Review by axw — APPROVED** [review]
thank you :)

**Pull request merged**